SPECS/libreswan.spec

```diff
@@ -20,66 +26,81 @@
     USE_LIBCURL=true \\\
     USE_LINUX_AUDIT=true \\\
     USE_NM=true \\\
+    USE_NSS_KDF=true \\\
     USE_SECCOMP=true \\\
-    USE_XAUTHPAM=true \\\
+    USE_AUTHPAM=true \\\
+    USE_DH2=true \\\
 %{nil}
 
-
 #global prever rc1
 
 Name: libreswan
 Summary: IPsec implementation with IKEv1 and IKEv2 keying protocols
 # version is generated in the release script
-Version: 3.26
-Release: %{?prever:0.}1.3%{?prever:.%{prever}}%{?dist}
+Version: 4.12
+Release: %{?prever:0.}2.1%{?prever:.%{prever}}%{?dist}.3
 License: GPLv2
 Url: https://libreswan.org/
+
 Source0: https://download.libreswan.org/%{?prever:with_development/}%{name}-%{version}%{?prever}.tar.gz
 %if 0%{with_cavstests}
 Source1: https://download.libreswan.org/cavs/ikev1_dsa.fax.bz2
 Source2: https://download.libreswan.org/cavs/ikev1_psk.fax.bz2
 Source3: https://download.libreswan.org/cavs/ikev2.fax.bz2
 %endif
 
-Patch1: libreswan-3.26-asn1-zu.patch
-
 Group: System Environment/Daemons
```

Comment:
Shouldn't this line go away, too?

Reply:
I wasn't sure about this one, I saw there are a lot of lines like theses in other packages (ovs has the same for example), so I guessed it could be good to keep it, but not sure if that brings anything to us. I can remove if you think it is better.

Reply:
It doesn't bring anything worth diverging from the upstream spec file, if anything at all.

Reply:
Removed.